pa11ycrawler/spiders/edx.py

```diff
@@ -308,8 +308,20 @@ def after_auto_auth(self, response):
                 errback=self.handle_error
             )
         else:
-            for url in self.start_urls:
-                yield self.make_requests_from_url(url)
+            yield scrapy.Request(
+                self.start_url,
+                callback=self.analyze_url_list,
```

Comment:
This is a little confusing to me, compared to the previous implementation. What if there are multiple urls? Can the callback take in a list object?